src/test/java/org/folio/innreach/controller/LocationMappingControllerTest.java

```diff
@@ -353,10 +378,18 @@ private static String baseMappingURL() {
     return baseMappingURL(PRE_POPULATED_CENTRAL_SERVER_ID, PRE_POPULATED_LIBRARY_ID);
   }
 
+  private static String baseMappingURLForAllLibraries() {
+    return baseMappingURLForAllLibraries(PRE_POPULATED_CENTRAL_SERVER_ID);
+  }
```

Comment:
just put the logic of `baseMappingURLForAllLibraries(String serverId)` in this method

Reply:
refactored